assignment.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "event_record.rb"
+event = {add: "1", month_view: "2", date_view: "3", all_events: "4", delete: "5", exit: "6"}  
+loop do 
+  puts
+  puts "For Adding an event, Enter: 1"
+  puts "For Viewing a specific month's event(s), Enter: 2"
+  puts "For Viewing a specific date's event(s), Enter: 3"
+  puts "For Viewing all events, Enter: 4"
+  puts "For Deleting an event, Enter: 5"
+  puts "To Quit, Enter 6"
+  line = gets.strip
+  events = []
+  count = -1
+  case line
+    when event[:add]
+      puts
+      check = "y"
+      while (check == "y")
+        count += 1
+        events << EventRecord.add_event  
+        puts "Enter name of the Event"
+        name = gets.strip
+        puts "Enter date for the event (yyyy:mm:dd)"
+        date = gets.strip
+        events[count].add_event_details(name, date)
```

Comment:
you don't need this count variable. push an object into array only when it's final.

Reply:
done